test/fhirpath.test.js

```diff
@@ -9,15 +9,15 @@ const _    = require('lodash');
 const items = fs.readdirSync(__dirname + '/cases/');
 
 // Set "focus" to true to turn on focus option
+const focus = false;
+const focusFile = /.*.yaml/;
+
 
 const endWith = (s, postfix) => {
   const idx = s.indexOf(postfix);
   return (idx > 0 && idx === s.length - postfix.length);
```

Comment:
If there are multiple occurrences of postfix in s? Maybe: s.length >= postfix.length && s.subtr(s.length - postifx.lenth) === postfix?

Reply:
Good catch.  Fixed.